tests/PhpAmqpLib/SslConnectionTest.php

```diff
@@ -97,7 +97,10 @@ public function it_returns_internal_connection()
     public function it_reconnects()
     {
         $connection = $this->createConnection();
+        $this->assertTrue($connection->isConnected());
+
         $connection->reconnect();
+        $this->assertTrue($connection->isConnected());
```

Comment:
Well okay, but it doesn't really tell the connection is `really` reconnected, if `reconnect` would do nothing at all, the tests would still pass. On the other hand I didn't had any assertions at all to begin with, so I'm good with that. Can we do those same things on ext-amqp?

Reply:
> Can we do those same things on ext-amqp?
For ext-amqp already like that.

\HumusTest\Amqp\AmqpExtension\ConnectionTest::it_reconnects
```
    /**
     * @test
     */
    public function it_reconnects()
    {
        $connection = $this->createConnection();

        $this->assertTrue($connection->isConnected());

        $connection->reconnect();

        $this->assertTrue($connection->isConnected());
    }
```